analyzers/tests/SonarAnalyzer.UnitTest/TestCases/EqualityOnFloatingPoint.CSharp11.cs

```diff
@@ -3,6 +3,31 @@
 using System.Runtime.InteropServices;
 
 public class EqualityOnFloatingPoint
+{
+    bool HalfEqual(Half first, Half second)
+        => first == second;    // Noncompliant {{Do not check floating point equality with exact values, use a range instead.}}
+    //           ^^
+
+    bool NFloatEqual(NFloat first, NFloat second)
+        => first == second;    // Noncompliant
+
```

Comment:
While touching this, you can fix all of these.. only because it's ugly
```suggestion
    bool NFloatEqual(NFloat first, NFloat second) =>
        first == second;    // Noncompliant

```